src/services/ApplicationService.ts

```diff
@@ -0,0 +1,97 @@
+import { S3 } from "aws-sdk";
+import config from "../config";
+import axios, { AxiosResponse } from "axios";
+import { Application } from "../models/application";
+import { randomUUID } from "crypto";
+import { validateApplicationObject } from "../validators/ApplicationValidator";
+import { getHeader } from "./AuthUtil";
+
+/**
+ * Checks if an S3 bucket exists.
+ * 
+ - @name checkBucket
+ - @param {S3} s3 - An instance of AWS S3 client.
+ - @param {string} bucket - The name of the S3 bucket to check.
+ - @returns {Promise<{success: boolean; message: string; data: object;}>} The result of the check operation.
+ */
+
+export const checkBucket = async (s3: S3, bucket:string | undefined) => { // This may not be needed as bucket will always exist
```

Comment:
1. This function should be called checkBucketExists
2. This function should return a boolean if it exists or if it doesn't
3. It is ok to thrown an error if bucket is undefined.

Reply:
Resolved